src/DataSource.ts

```diff
@@ -56,13 +56,17 @@ export class DataSource extends DataSourceWithBackend<TimestreamQuery, Timestrea
       return query;
     }
 
+    // create a copy of scopedVars without $__interval_ms for using with rawQuery
+    var scopedVarsWithoutTimestamp = Object.assign({}, scopedVars);
+    delete scopedVars.__interval_ms;
```

Comment:
Maybe something like:
```suggestion
    const {__interval_ms, __interval, ...queryScopedVars} = scopedVars;
```